scripts/update-index.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# This script update 'docs/index.md' with 'README.md' 
+#  changing github/learn-ocaml/docs URLs to relative path.
+
+readme="$PWD/README.md"
+index="$PWD/docs/index.md"
```

Comment:
Beware that `$PWD/README.md` is not necessarily an absolute path.

To use an absolute path w.r.t. the exact path of the `.sh` script itself (so that the script is robust modulo `cd` commands), I can suggest to write this Bash invocation:

```
srcdir=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd)

readme=$(readlink -f "$srcdir/../README.md")
index=$(readlink -f "$srcdir/../docs/index.md")
```